docs/xshareclasses.md

```diff
@@ -419,7 +419,7 @@ behavior, which can improve the performance of class loading from the shared cla
 
         -Xshareclasses:nonpersistent
 
-:   Uses a non-persistent cache. The cache is lost when the operating system shuts down. Non-persistent and persistent caches can have the same name. On Linux, macOS, and Windows systems, you must always use the `nonpersistent` suboption when you run utilities such as `destroy` on a non-persistent cache.
+:   Uses a non-persistent cache. The cache is lost when the operating system shuts down. Non-persistent and persistent caches can have the same name. On Linux, macOS, Windows, and z/OS systems, you must always use the `nonpersistent` suboption when you run utilities such as `destroy` on a non-persistent cache.
```

Comment:
We don't need to mention z/OS here since nonpersistent is the default.

Reply:
We should have the opposite comment for z/OS under 'persistent'. I see under `persistent` we mention "On AIX, ...", but this is long obsolete and can be changed to "On z/OS".